### PR TITLE
Removed some extra lines #1571

### PR DESCRIPTION
**Pull request by j0rt3g4**
The easiest way is just to use the PS Script and it's still the recommended process, the supposed scenario is that they are just inside Azure AD and not other AD that manages policies or administrative templates.

### CODE REVIEW

**Comment by j0rt3g4** [comment]
Copy review @nenonix @JohanFreelancer9 is required on this one guys. Thank you in advance!

File: OneDrive/use-silent-account-configuration.md [hunk 1]
```diff
@@ -35,21 +35,13 @@ Before you can enable silent account configuration, you need to join your device
...
-    
 - Windows Server 2016 
-    
 - Windows Server 2012 R2 
-    
 - Windows Server 2012 
-    
 - Windows Server 2008 R2
-
 If you federate your on-premises Active Directory with Azure AD, you must use AD FS to enable this feature. For info about using Azure AD Connect, see [Getting started with Azure AD Connect using express settings](/azure/active-directory/hybrid/how-to-connect-install-custom).
```
Comment (JohanFreelancer9) [inline OneDrive/use-silent-account-configuration.md]: Please check formatting in rich view. Line 44 noflows into Line 45.

Comment (j0rt3g4) [reply OneDrive/use-silent-account-configuration.md]: I do the editions on Visual Studio, I was not able to add or see that line, and I'm unable to apply your suggestion directly from the text. I've added some lines and remove it, let me know if this conversation can be marked as resolved. Thank you in advance

Comment (JohanFreelancer9) [reply OneDrive/use-silent-account-configuration.md]: @j0rt3g4 It is perfect now, thank you. It can be marked as resolved.

Comment (j0rt3g4) [reply OneDrive/use-silent-account-configuration.md]: (Y) thank you @JohanFreelancer9

**Comment by scanum** [comment]
@kaarins PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Pull request merged**